webots_ros2_core/webots_ros2_core/utils.py

```diff
@@ -135,10 +137,9 @@ def get_webots_version_major_number():
     return 0
 
 
-def get_webots_version(path=None):
+def get_webots_version(path):
     """Webots version as a string."""
-    versionFile = os.path.join(path if path is not None else get_webots_home(),
-                               'resources', 'version.txt')
+    versionFile = os.path.join(path, 'resources', 'version.txt')
```

Comment:
Now, I believe this patch is not needed any more since there is no risk of recursion and it looks better to be able to call `get_webots_version()` without any parameter.